### DISCUSSION TITLE
FIP/FOP requirement for non-fuel units

### DISCUSSION THREAD

**Discussion by ZekeCG** [ercot/api-specs#47] [General]
I am submitting TPOs (Three Part Offers) via the SOAP API for BESS units, until now I've been including the FIP/FOP with parameter of 0. Would it be possible to simply omit these for batteries? They do not provide any real value since the live prices of natural gas and fuel oil are meaningless to batteries. Any help is greatly appreciated.

**Comment by sjinright-ercot** [comment]
You will need to submit FIP/FOP values for the resources when either submitting new or updating EOC past operating period.  That would mean using 0's for batteries. FIP/FOP for the hours in the operating period are not allowed though.
